src/libAtomVM/globalcontext.h

```diff
@@ -83,7 +83,7 @@ typedef struct
     uint64_t ref_ticks;
 
     void *platform_data;
-} GlobalContext;
+};
```

Comment:
I don't have an issue with this, but I am curious how you can remove the typedef here but still use it below without `struct`.  Is it typedef'd elsewhere?

Reply:
You're right, there is already somewhere a typedef, but I might repeat it for consistency.